shared-helpers/src/locales/general.json

```diff
@@ -488,7 +488,25 @@
   "authentication.timeout.action": "Stay logged in",
   "authentication.timeout.signOutMessage": "We care about your security. We logged you out due to inactivity. Please sign in to continue.",
   "authentication.timeout.text": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information and be logged out if you choose not to respond.",
+  "basicsCard.sectionTitle": "Learn more about the basics of affordable housing",
```

Comment:
Is there a reason these translations need to be in this file and not the `page_content/locale_overrides` file? All translation keys that are only in Detroit should be in the `locale_overrides` folder so that this file basically matches core